Kernel/API/KeyCode.h

```diff
@@ -163,7 +165,7 @@ enum KeyModifier {
     Mod_Shift = (1 << 2),
     Mod_Super = (1 << 3),
     Mod_AltGr = (1 << 4),
-    Mod_Keypad = (1 << 5),
+    Mod_Keypad = (1 << 4),
```

Comment:
Keypad should still be `(1 << 5)`, right?

Reply:
Whoops, yes, I broke this while amending. Fixed.